controller/Dockerfile.proto

```
@@ -3,15 +3,15 @@ FROM --platform=$BUILDPLATFORM mcr.microsoft.com/oss/go/microsoft/golang@sha256:
 
 LABEL Name=retina-builder Version=0.0.1
 
-RUN apt-get update &&\
-    apt-get -y install lsb-release wget software-properties-common gnupg file git make unzip
+RUN yum update -y &&\
+    yum install -y unzip
```

on mariner, use `tdnf` not `yum`
```suggestion
RUN tdnf update &&\
    tdnf install -y unzip
```